macros/print_base/start_print.cfg

```diff
@@ -112,6 +114,8 @@ gcode:
         RESPOND MSG="Extruder temperature OK"
     {% endif %}
 
+    # perform homing after hotend heating
+    G28 Z
```

Comment:
Why not anymore after the nozzle clean ?
I think it's better to do it as late as possible to give it as much time as possible to stabilize. Also, I think it's better to do it after a nozzle clean as it's more stable without any remaining filament. Don't you think ?

Reply:
Wow, is that already happened to you ?
But Ok, then we add this homing after the heating of the nozzle.

But maybe we also need to add another G28Z after the clean nozzle ? If we don't do that, we will have a lot of different Z offsets computed by the Z calibration plugin as the offset will be applied depending of how much filament there was on the nozzle I think...

Reply:
Yes it's arrived with a 0.8mm nozzle, i had a fat cold blob of plastic on the nozzle.

It's not necessary to add G28 Z after cleaning, we will be a little higher but it's not important, z-calibration don't take care of it

Reply:
Mhmm Ok let's try it and if I have trouble with the offset (I like to get consistent offsets between prints to check everything is alright just looking at the number), I'll add another home Z.